src/elements/flip-card/flip-card-details/flip-card-details.ts

```diff
@@ -32,25 +40,6 @@ export class SbbFlipCardDetailsElement extends LitElement {
       .forEach((el: Element) => el.setAttribute(cardFocusableAttributeName, ''));
   }
 
-  public override connectedCallback(): void {
-    super.connectedCallback();
-    this._checkForSlottedActions();
```

Comment:
Is the call to _checkForSlottedActions no longer necessary?

Reply:
yes, it's not needed as the observer does also check whenever its connected